cmd/kubectl-gadget/profile/cpu.go

```diff
@@ -148,7 +145,7 @@ func (p *CPUParser) DisplayResultsCallback(traceOutputMode string, results []str
 			}
 		}
 
-		p.PrintColumnsHeader()
+		p.BuildColumnsHeader()
```

Comment:
It's missing the logic to print the returned value.

Reply:
Managed in https://github.com/kinvolk/inspektor-gadget/pull/762